gui/theme/common/languages/de.xml

```diff
@@ -186,15 +186,15 @@
 		<string name="change_fs_for_hdr">Dateisystem für %tw_partition_name% ändern</string>
 		<string name="change_fs_for_hdr_s">Partition: %tw_partition_name% &gt; Dateisystem auswählen</string>
 		<string name="change_fs_warn1">Einige ROMs oder Kernel unterstützen möglicherweise manche</string>
-		<string name="change_fs_warn2">Dateisysteme nicht. Überlegt handeln!</string>
+		<string name="change_fs_warn2">Dateisysteme nicht. Bitte nochmals überprüfen!</string>
 		<string name="change_fs_confirm">%tw_partition_name% ändern?</string>
 		<string name="formatting">Formatierung...</string>
 		<string name="format_complete">Formatieren abgeschlossen</string>
 		<string name="swipe_change_fs">Dateisystem ändern</string>
 		<string name="swipe_change_s">Ändern</string>
 		<string name="back_btn">Zurück</string>
 		<string name="wipe_enc_btn">Verschlüsselung entfernen</string>
-		<string name="swipe_factory_reset">Werkseinstellungen herstellen</string>
+		<string name="swipe_factory_reset">Zurücksetzen</string>
```

Comment:
Depends on how much space you actually have. If `Auf Werkseinstellungen zurücksetzen` does not get truncated, that would be better.